app/pages/item/additionalReference/index/AdditionalReferencePage.scala

```diff
@@ -30,7 +30,7 @@ case class AdditionalReferencePage(itemIndex: Index, additionalReferenceIndex: I
 
   override def path: JsPath = AdditionalReferenceSection(itemIndex, additionalReferenceIndex).path \ toString
 
-  override def toString: String = "additionalReference"
+  override def toString: String = "addAdditionalReference"
```

Comment:
Can this be reverted please as this will affect the submission logic and current drafts if deployed